wremnants/datasets/datagroupsLowPU.py

```diff
@@ -20,81 +17,86 @@ def __init__(self, infile, combine=False, flavor=""):
             ),
             DYee=dict(
                 members=[self.datasets["DYee"]],
                 label=r"DY #rightarrow e^{#plus}e^{#minus} (MiNNLO)",
                 color=ROOT.TColor.GetColor(248, 206, 104),
-                signalOp = signalOp,
-            ),
-            SingleMuon=dict(
-                members=[self.datasets["singlemuon"]],
-                label="Data",
-                color=ROOT.kBlack,
-                signalOp = signalOp if flavor == "mumu" else None,
-            ),
-            SingleElectron=dict(
-                members=[self.datasets["singleelectron"]],
-                label="Data",
-                color=ROOT.kBlack,
-                signalOp = signalOp if flavor == "ee" else None,
+                signalOp = None,
             ),
             
-            
-            ## individual procs
-            
+            # individual procs
             DYtautau=dict(
                 members = [self.datasets[x] for x in ["DYtautau"]],
                 label="DYtautau",
                 color=ROOT.TColor.GetColor(100, 192, 232),
-                signalOp = signalOp,
+                signalOp = None,
             ),
             WJets=dict(
                 members = [self.datasets[x] for x in ["WplusJetsToMuNu", "WminusJetsToMuNu", "WplusJetsToTauNu", "WminusJetsToTauNu", "WplusJetsToENu", "WminusJetsToENu"]],
                 label="WJets",
                 color=ROOT.TColor.GetColor(100, 192, 232),
-                signalOp = signalOp,
+                signalOp = None,
             ),
             WZTo3LNu=dict(
                 members = [self.datasets[x] for x in ["WZTo3LNu"]],
                 label="WZTo3LNu",
                 color=ROOT.TColor.GetColor(100, 192, 232),
-                signalOp = signalOp,
+                signalOp = None,
             ),
             WWTo2L2Nu=dict(
                 members = [self.datasets[x] for x in ["WWTo2L2Nu"]],
                 label="WWTo2L2Nu",
                 color=ROOT.TColor.GetColor(100, 192, 232),
-                signalOp = signalOp,
+                signalOp = None,
             ),
             ZZ=dict(
                 members = [self.datasets[x] for x in ["ZZ"]],
                 label="ZZ",
                 color=ROOT.TColor.GetColor(100, 192, 232),
-                signalOp = signalOp,
+                signalOp = None,
             ),
             
         )
         
-
-    
+        # data
+        if flavor == "mumu":
+            self.groups.update(
+                SingleMuon=dict(
+                    members=[self.datasets["singlemuon"]],
+                    label="Data",
+                    color=ROOT.kBlack,
+                    signalOp = None,
+                ),
+            )
+        if flavor == "ee":
+            self.groups.update(
+                SingleElectron=dict(
+                    members=[self.datasets["singleelectron"]],
+                    label="Data",
+                    color=ROOT.kBlack,
```

Comment:
Maybe you could use the hex color here and in ROOT so that it works with both ROOT and matplotlib